src/commands/stats.py

```diff
@@ -0,0 +1,132 @@
+# file to store messages being dynamically updated between reboots
+import json
+from typing import List, NamedTuple, Dict, Any, Union, TYPE_CHECKING
+
+import discord
+from discord import app_commands
+
+from src.constants import VOLUNTEER_ROLE
+
+if TYPE_CHECKING:
+    from src.bot import BotClient
+
+SUBSCRIBE_MSG_FILE = 'subscribed_messages.json'
+
+
+class SubscribedMessage(NamedTuple):
+    """A message that is updated when the server statistics change."""
+
+    channel_id: int
+    message_id: int
+    members: bool = True
+    warnings: bool = True
+    stats: bool = False
+
+    @classmethod
+    def load(cls, dct: Dict[str, Any]) -> Union['SubscribedMessage', Dict[str, Any]]:
+        """Load a SubscribedMessage object from a dictionary."""
+        if tuple(dct.keys()) == cls._fields:
+            return cls(
+                dct['channel_id'],
+                dct['message_id'],
+                dct['members'],
+                dct['warnings'],
+                dct['stats']
+            )
+        return dct
```

Comment:
Perhaps we should downgrade to a splat and let it throw errors if doesn't have the required keys. Also simplifies the return type.
```suggestion
        return cls(**dct)
```

Reply:
8792a30